src/main/java/com/robinpowered/sdk/RobinApi.java

```diff
@@ -134,6 +135,16 @@ public boolean hasCredential() {
     }
 
     /**
+     * Returns whether a the interface is authenticated with a valid credential.
+     * Valid credentials have non-null values.
+     *
+     * @return Whether a valid credential has been set.
+     */
+    public boolean isAuthenticated() {
+        return credential != null && credential.getValue() != null;
```

Comment:
If we were to simply expose the credential by offering a getter method, that would be easiest!
Is this the path you'd like to take? I had assumed there were reasons we were keeping the authentication credential private and non-exposed